Week3/QA_PREP_EXERCISES.md

```diff
@@ -0,0 +1,17 @@
+As a preparation step for the upcoming Q&A, you need to work on the following exercise, which is based on the prep exercise of the previous week. 
+
+### Exercise
+
+Last week you created an ERD for the database for storing food recipes.
+How can you normalize your database based on what you learned this week?
```

Comment:
What are you expecting them to do here? Answer the question or produce a database?

Reply:
Mainly to answer the question , and in order to answer it, they should produce old and new database.  I will clarify in the question